fluent-sequence/src/map_async.ts

```diff
@@ -0,0 +1,52 @@
+import { FluentBundle } from "@fluent/bundle";
+
+export function mapBundleAsync(
+  bundles: AsyncIterable<FluentBundle>,
+  ids: string
+): Promise<FluentBundle | null>
+
+export function mapBundleAsync(
+  bundles: AsyncIterable<FluentBundle>,
+  ids: Array<string>
+): Promise<Array<FluentBundle | null>>;
```

Comment:
Is there documentation on what these do, in combination with the impl below??

Reply:
They're function overloads: https://www.typescriptlang.org/docs/handbook/functions.html#overloads. They allow specifying extra typing constraints when the implementation is quite loose. E.g. "if you pass an array, expect an array as a result".